collect_app/src/main/AndroidManifest.xml

```diff
@@ -111,7 +111,7 @@ the specific language governing permissions and limitations under the License.
             android:windowSoftInputMode="stateHidden" />
         <activity android:name=".activities.InstanceChooserList" />
         <activity
-            android:name=".activities.FillBlankFormActivity"
+            android:name=".formlists.blankformlist.BlankFormListActivity"
             android:launchMode="singleTop" />
```

Comment:
I don't know why we have this. Maybe to avoid having duplicate activities when a user opens it directly fro an external app?  But then it should be added in all activities that are exported... Do you think we should get rid of it?

Reply:
Does seem a little risky to kill now on top of all the other changes... Let's leave it.